app/models/doi.rb

```diff
@@ -1874,7 +1874,7 @@ def metadata_version
   end
 
   def current_media
-    media.order("media.created DESC").first
```

Comment:
This is already defined in the association `has_many :media, -> { order "created DESC" }..` as a result it produces a query like this `"SELECT `media`.* FROM `media` WHERE `media`.`dataset` = 38 ORDER BY created DESC, media.created DESC"`